docs/docusaurus.config.js

```diff
@@ -0,0 +1,92 @@
+const remarkSources = require('remark-sources');
+const remarkCollapse = require('remark-collapse'); // TODO: probably won't need this one
+const { remarkProgramOutput } = require("./plugins/program_output");
+
+let versions = [];
+try {
+  versions = require('./versions.json');
+} catch (ex) {
+  // Nothing to do here, in dev mode, only
+  // one version of the doc is available
+}
+let legacyVersions = [];
+
+module.exports = {
+  title: 'Rasa SDK Documentation',
+  tagline: 'Rasa SDK Documentation',
+  // TODO: is it needed?
+  url: 'http://something.com',
+  baseUrl: '/docs/rasa/rasa-sdk/',  //TODO: seems fake
```

Comment:
So I'm not sure how these docs are actually going to get deployed -- we didn't clear up whether they were going to be accessible from the rasa open source sidebar (in which case it would make sense to have them under the rasa docs) or if they're going to have their own header bar and be completely separate

Reply:
ok let's keep it like that for now 👍